### DISCUSSION TITLE
Dedicated template file per page

### DISCUSSION THREAD

**Discussion by xldrkp** [squidfunk/mkdocs-material#2837] [Q&A] [answered]
Hi!

Is it possible to build dedicated template files for the use with certain pages? Like saying

```yaml
----
layout: team
----
```

in the front matter of an `index.md` and thus using `team.html` from the `overrides` folder?

**Comment by squidfunk** [comment, marked as answer]
Yes:

``` yaml
----
template: team.html
----
```

**Reply by xldrkp** [reply]
Great, that's what I was looking for! Thankyou very much!
